assets/js/banCtrl.js

```diff
@@ -33,28 +33,33 @@ define([
                 return;
             }
 
-            if ($scope.banInfo.banNote.length > 300) {
-                $scope.banError = "The ban note cannot be longer than 300 characters.";
+            if ($scope.banInfo.username.substring(0,3) === '/u/') {
```

Comment:
I guess we can add that case later. Presumably mods will be able to figure out what the error is after being told the username is invalid, so it's not a huge issue.